### PR TITLE
new(gatsby-plugin-vanilla-extract): add to repo

### PR DESCRIPTION
**Pull request by marvinjude**
## Description

Add [gatsby-plugin-vanilla-extract](https://github.com/KyleAMathews/gatsby-plugin-vanilla-extract) to repo

### CODE REVIEW

File: packages/gatsby-plugin-vanilla-extract/README.md
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/README.md:7]:
````suggestion
```shell
npm install gatsby-plugin-vanilla-extract @vanilla-extract/babel-plugin @vanilla-extract/css @vanilla-extract/webpack-plugin
```
````

Always better to put install messages in big code blocks

File: packages/gatsby-plugin-vanilla-extract/README.md
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/README.md:19]:
```suggestion
See the [vanilla-extract website](https://vanilla-extract.style/) for full documentation.
```

File: packages/gatsby-plugin-vanilla-extract/README.md
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/README.md:23]:
```suggestion
`src/pages/index.tsx`:
```

File: packages/gatsby-plugin-vanilla-extract/README.md
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/README.md:35]:
```suggestion
`src/styles/index.css.ts`:
```

File: packages/gatsby-plugin-vanilla-extract/README.md
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/README.md:47]:
```suggestion
### Example Project

Kick off your project using Gatsby's [example project](TODO)
```

That's the example you'll be creating in the monorepo

File: packages/gatsby-plugin-vanilla-extract/README.md
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/README.md:16]:
````suggestion

You can also pass any [vanilla-extract configuration object](https://vanilla-extract.style/documentation/setup/#configuration) to the plugin.

```js
module.exports = {
  plugins: [
    {
      resolve: `gatsby-plugin-vanilla-extract`,
      options: {
        identifiers: `short`
      }
    }
  ],
};
```

````

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/package.json:3]: @moonmeister Needs to chime in here but I think the correct way of doing this would be use the old current latest version (so `1.0.6`) and then adding a changeset that has the major change in it.

Also @KyleAMathews you'll need to connect with @moonmeister regarding npm credentials

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/package.json:11]: URL incorrect

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/package.json:20]: URL incorrect

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/package.json:22]: URL incorrect

File: packages/gatsby-plugin-vanilla-extract/renovate.json
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/renovate.json:1]: You'll want to remove this file + yarn.lock file

**Review by LekoArts — CHANGES_REQUESTED** [review]

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (moonmeister) [reply packages/gatsby-plugin-vanilla-extract/package.json:3]: Correct. Leave the current version and you can add a changeset to bump.

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (moonmeister) [reply packages/gatsby-plugin-vanilla-extract/package.json:11]: See the contributor docs for information on this formatting. Or another plugin.

**Comment by moonmeister** [comment]
Thanks for opening this! Before this is merged We'll need to get @KyleAMathews to transfer the existing repo transferred to the GUC and add myself as a maintainer on the npm package.

File: packages/gatsby-plugin-vanilla-extract/LICENSE
Comment (moonmeister) [inline packages/gatsby-plugin-vanilla-extract/LICENSE:1]: this can be removed as it is the same as the repository license.

**Review by moonmeister — CHANGES_REQUESTED** [review]

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (marvinjude) [reply packages/gatsby-plugin-vanilla-extract/package.json]: @moonmeister Is it okay to run the `cs` script for this?

I ran it and the changes in the [last commit](https://github.com/gatsby-uc/plugins/pull/148/commits/907876dfbb49d2d7e8d2db743c5455f1a7569750) got added

File: packages/gatsby-plugin-vanilla-extract/.gitignore
Comment (marvinjude) [reply packages/gatsby-plugin-vanilla-extract/.gitignore]: Sure

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (moonmeister) [reply packages/gatsby-plugin-vanilla-extract/package.json]: yup. looks good.

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (LekoArts) [inline packages/gatsby-plugin-vanilla-extract/package.json:28]:
```suggestion
    "@vanilla-extract/webpack-plugin": "^2.0.0"
```

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (moonmeister) [inline packages/gatsby-plugin-vanilla-extract/package.json:7]: This is breaking tests.

**Review by moonmeister — CHANGES_REQUESTED** [review]

File: packages/gatsby-plugin-vanilla-extract/package.json
Comment (marvinjude) [reply packages/gatsby-plugin-vanilla-extract/package.json]: I think we're all set now

File: .changeset/big-coins-smoke.md
Comment (LekoArts) [inline .changeset/big-coins-smoke.md:5]:
````suggestion
This moves the repository from [KyleAMathews/gatsby-plugin-vanilla-extract](https://github.com/KyleAMathews/gatsby-plugin-vanilla-extract) to [gatsby-uc/plugins](https://github.com/gatsby-uc/plugins) to help with better maintenance on the plugin itself.

There was also a breaking change made to the plugin: The `vanilla-extract` dependencies are now marked as `peerDependencies` and not bundled with the plugin anymore.

In order to migrate, run the following in your terminal:

```shell
npm install gatsby-plugin-vanilla-extract@latest @vanilla-extract/babel-plugin @vanilla-extract/css @vanilla-extract/webpack-plugin
```
````

Comment (marvinjude) [reply .changeset/big-coins-smoke.md]: Thanks!

**Review by moonmeister — APPROVED** [review]
Just waiting for NPM access and the old repo to be transfered.

**Review by LekoArts — APPROVED** [review]

**Comment by KyleAMathews** [comment]
Got this error when trying to transfer the repo:

`You don’t have the permission to create public repositories on gatsby-uc`

**Comment by KyleAMathews** [comment]
Screen Shot 2022-02-01 at 9 04 29 AM

**Comment by moonmeister** [comment]
> Got this error when trying to transfer the repo:
> 
> `You don’t have the permission to create public repositories on gatsby-uc`

Added you to the org, you should be good now.

**Comment by KyleAMathews** [comment]
Great! Moved the repo over now as well 👍

**Comment by moonmeister** [comment]
Thanks all!

**Pull request merged**
